### PR TITLE
Replace instances of SUM(CASE WHEN ...) with COUNT_IF()

### PR DESCRIPTION
**Pull request by timblair**
Replace instances of `SUM(CASE WHEN ...)` with `COUNT_IF()` after @kevindew's [excellent suggestion][comment].

[comment]: https://github.com/alphagov/govuk-developer-docs/pull/2977#discussion_r581070710

### CODE REVIEW

File: source/manual/query-cdn-logs.html.md [hunk 1]
```diff
@@ -332,7 +325,7 @@ SELECT
     ROUND(APPROX_PERCENTILE(request_time, 0.95), 3) AS p95,
     ROUND(MAX(request_time), 3) AS p100
 FROM fastly_logs.govuk_www
-WHERE year = 2021 AND month = 2 AND date = X
+WHERE year = 2020 AND month = 12 AND date = 19
```
Comment (kevindew) [inline source/manual/query-cdn-logs.html.md:328]: Did you mean to replace the X here? I notice it was added in on some of the earlier queries such as https://github.com/alphagov/govuk-developer-docs/pull/2978/files#diff-76282ecd5f5d2c52fa5fc1d72876392a5547857ddf7688fe18c6ca03ce8c2af6R203. There's a couple more instances of this.

**Review by kevindew — COMMENTED** [review]
Nice - that's a heck of a lot of `THEN 1 ELSE 0 END`'s removed 👍

File: source/manual/query-cdn-logs.html.md (hunk 1 above)
Comment (timblair) [reply source/manual/query-cdn-logs.html.md]: Ah, that's an artefact of copying-and-pasting from a somewhere else. It's not really important, but I've pushed a change so things that don't need to be changed are left alone.

**Review by kevindew — APPROVED** [review]
Nice one, thanks

**Pull request merged**